docs/openapi.yaml

```diff
@@ -5,59 +5,66 @@ info:
   license:
     name: MIT
 paths:
-  /lms:
-    post:
+  /lms/lti/canvas/{serviceId}:
+    parameters:
+      - in: path
+        name: serviceId
```

Comment:
Say something about valid values. Maybe even create a string enum for the valid values.

Reply:
added enums, with nullable: true